flink-runtime-web/web-dashboard/src/app/pages/job/checkpoints/job-checkpoints.component.html

```diff
@@ -550,6 +550,19 @@
               {{ checkPointConfig['checkpoints_after_tasks_finish'] ? 'Enabled' : 'Disabled' }}
             </td>
           </tr>
+          <tr>
+            <td>Changelog</td>
```

Comment:
I think the description of `Changelog state-backend` looks better.

Reply:
Fixed.
![image](https://user-images.githubusercontent.com/18653940/167767997-be5d7fef-e92d-41d6-a6c7-e1ffc6fd1ab7.png)